plugins/processors/awsapplicationsignals/internal/attributes/attributes.go

```diff
@@ -26,4 +26,7 @@ const (
 	ResourceDetectionHostId   = "host.id"
 	ResourceDetectionHostName = "host.name"
 	ResourceDetectionASG      = "ec2.tag.aws:autoscaling:groupName"
+
+	// ApplicationSignals behaviour-changing attributes
+	AwsApplicationSignalsMetricResourceKeys = "aws.application_signals.metric_resource_keys"
```

Comment:
nit: capitalization of AWS
```suggestion
	AWSApplicationSignalsMetricResourceKeys = "aws.application_signals.metric_resource_keys"
```

Reply:
Done